java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -174,6 +165,32 @@ open class DevToolsService : Service() {
         )
     }
 
+    private fun createAndSendProxyMessages(
+        actualMessage: ProxyMessage<CrdtData, CrdtOperation, Any?>,
+        storeType: String
+    ) {
+        when (actualMessage) {
+            is ProxyMessage.SyncRequest -> {
+                val syncMessage = StoreSyncMessage(
+                    JsonValue.JsonNumber(actualMessage.id?.toDouble() ?: 0.0)
+                )
+                devToolsServer.send(syncMessage.toJson())
+            }
+            is ProxyMessage.Operations -> {
+                val storeMessage = StoreOperationMessage(actualMessage, storeType)
+                devToolsServer.send(storeMessage.toJson())
+            }
+            is ProxyMessage.ModelUpdate -> {
+                val storeMessage = ModelUpdateMessage(actualMessage, storeType)
+                devToolsServer.send(storeMessage.toJson())
+            }
+        }
```

Comment:
Minor nit, this could be simplified a little by using the `when` as an expression:

```suggestion
        val storeMessage = when (actualMessage) {
            is ProxyMessage.SyncRequest ->
                StoreSyncMessage(
                    JsonValue.JsonNumber(actualMessage.id?.toDouble() ?: 0.0)
                )
            is ProxyMessage.Operations ->
                StoreOperationMessage(actualMessage, storeType)
            is ProxyMessage.ModelUpdate ->
                ModelUpdateMessage(actualMessage, storeType)
        }
        devToolsServer.send(storeMessage)
```